### PR TITLE
Update Japanese translation (LINE 400-449)

### PR DESCRIPTION
**Pull request by yosshy**
/hold for japanese developer's review
/assign @t-inu
/cc @atoato88 @s-kawamura-w664

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -397,61 +397,56 @@
...
-	"Paused {{.count}} containers in: {{.namespaces}}": "次のnamespaceに存在する {{.count}} 個のコンテナを停止しました: {{.namespaces}}",
-	"Pausing node {{.name}} ...": "ノード {{.name}} を一時停止しています ...",
-	"Pausing node {{.name}} ... ": "",
-	"Please also attach the following file to the GitHub issue:": "",
-	"Please create a cluster with bigger disk size: `minikube start --disk SIZE_MB` ": "",
-	"Please either authenticate to the registry or use --base-image flag to use a different registry.": "",
-	"Please enter a value:": "",
-	"Please free up disk or prune images.": "",
-	"Please increse Desktop's disk size.": "",
+	"Number of extra disks created and attached to the minikube VM (currently only implemented for hyperkit and kvm2 drivers)": "作成して minikube VM に接続する追加ディスク数 (現在、kyperkit と kvm2 ドライバーでのみ実装されています)",
```
Comment (atoato88) [inline translations/ja.json:421]:
```suggestion
	"Number of extra disks created and attached to the minikube VM (currently only implemented for hyperkit and kvm2 drivers)": "作成して minikube VM に接続する追加ディスク数 (現在、hyperkit と kvm2 ドライバーでのみ実装されています)",
```

File: translations/ja.json [hunk 2]
```diff
@@ -397,61 +397,56 @@
...
+	"Overwrite image even if same image:tag name exists": "同じ image:tag 名が存在していてもイメージを上書きします",
+	"Path to the Dockerfile to use (optional)": "使用する Dockerfile へのパス (任意)",
+	"Pause": "停止",
```
Comment (atoato88) [inline translations/ja.json:440]: I think "停止" is "Stop" in English, is it better "一時停止" here?
WDYT?

File: translations/ja.json [hunk 3]
```diff
@@ -397,61 +397,56 @@
...
+	"Paused {{.count}} containers": "{{.count}} 個のコンテナーを停止しました",
```
Comment (atoato88) [inline translations/ja.json:441]: ditto.

File: translations/ja.json [hunk 4]
```diff
@@ -397,61 +397,56 @@
...
+	"Paused {{.count}} containers in: {{.namespaces}}": "{{.namespaces}} に存在する {{.count}} 個のコンテナーを停止しました",
```
Comment (atoato88) [inline translations/ja.json:442]: ditto.

**Comment by atoato88** [comment]
@yosshy 
I left some comments.
Please check those.

**Review by s-kawamura-w664 — COMMENTED** [review]
Please check my comments.

File: translations/ja.json [hunk 5]
```diff
@@ -397,61 +397,56 @@
...
-	"None of the known repositories is accessible. Consider specifying an alternative image repository with --image-repository flag": "既知のいずれのリポジトリにもアクセスできません。--image-repository フラグとともに代替のイメージ リポジトリを指定することを検討してください",
-	"Noticed you have an activated docker-env on {{.driver_name}} driver in this terminal:": "",
-	"Noticed you have an activated podman-env on {{.driver_name}} driver in this terminal:": "",
+	"None of the known repositories are accessible. Consider specifying an alternative image repository with --image-repository flag": "アクセス可能な既知リポジトリーはありません。--image-repository フラグを用いた代替イメージリポジトリー指定を検討してください",
+	"None of the known repositories in your location are accessible. Using {{.image_repository_name}} as fallback.": "ロケーション内でアクセス可能な既知リポジトリーはありません。フォールバックとして {{.image_repository_name}} を使用します。",
+	"Noticed you have an activated docker-env on {{.driver_name}} driver in this terminal:": "通知: この端末で {{.driver_name}} ドライバーの docker-env を有効化する必要があります:",
+	"Noticed you have an activated podman-env on {{.driver_name}} driver in this terminal:": "通知: この端末で {{.driver_name}} ドライバーの podman-env を有効化する必要があります:",
```
[the remark this answers is not in the document]
Comment (yosshy) [reply translations/ja.json]: I will update it as your suggest.

File: translations/ja.json [hunk 6]
```diff
@@ -397,61 +397,56 @@
...
...users: [1] zsh completions are only supported in versions of zsh \u003e= 5.2\n\tNote for fish users: [2] please refer to this docs for more details https://fishshell.com/docs/current/#tab-completion\n": "指定されたシェル用の minikube シェル補完コマンドを出力 (bash、zsh、fish)\n\n\tbash-completion バイナリーに依存しています。インストールコマンドの例:\n\tOS X:\n\t\t$ brew install bash-completion\n\t\t$ source $(brew --prefix)/etc/bash_completion\n\t\t$ minikube completion bash \u003e ~/.minikube-completion  # bash ユーザー用\n\t\t$ minikube completion zsh \u003e ~/.minikube-completion  # zsh ユーザー用\n\t\t$ source ~/.minikube-completion\n\t\t$ minikube completion fish \u003e ~/.config/fish/completions/minikube.fish # fish ユーザー用\n\tUbuntu:\n\t\t$ apt-get install bash-completion\n\t\t$ source /etc/bash_completion\n\t\t$ source \u003c(minikube completion bash) # bash ユーザー用\n\t\t$ source \u003c(minikube completion zsh) # zsh ユーザー用\n\t\t$ minikube completion fish \u003e ~/.config/fish/completions/minikube.fish # fish ユーザー用\n\n\tさらに、補完コマンドをファイルに出力して .bashrc 中で source したいかもしれません\n\n\t注意 (zsh ユーザー): [1] zsh 補完コマンドは zsh バージョン \u003e= 5.2 でのみサポートしています\n\t注意 (fish ユーザー): [2] 詳細はこちらのドキュメントを参照してください https://fishshell.com/docs/current/#tab-completion\n",
```
[the remark this answers is not in the document]
Comment (yosshy) [reply translations/ja.json]: done.

File: translations/ja.json (hunk 2 above)
Comment (yosshy) [reply translations/ja.json]: Looks good.

File: translations/ja.json (hunk 4 above)
Comment (yosshy) [reply translations/ja.json]: done.

File: translations/ja.json (hunk 3 above)
Comment (yosshy) [reply translations/ja.json]: done.

**Review by yosshy — COMMENTED** [review]
@atoato88 @t-inu thank you for your review.

**Comment by t-inu** [comment]
/lgtm
I confirmed all the fixes. It's OK.

**Comment by yosshy** [comment]
@t-inu thank you for your check.

/unhold
/assign @afbjorklund 
Japanese reviews have finished. Please merge this.

**Comment by yosshy** [comment]
@spowelljr thank you.

**Pull request merged**